docs/network/network_binding_plugins.md

```diff
@@ -69,7 +69,6 @@ A network binding plugin configuration consist of the following steps:
   - Binding CNI plugin.
   - Binding NetworkAttachmentDefinition manifest.
   - Access to the sidecar image.
-  - Enable `NetworkBindingPlugins` Feature Gate (FG).
```

Comment:
Commit message:
Please consider stating that the network binding plugins are GAed since v1.5.
https://github.com/kubevirt/kubevirt/pull/13314